src/components/character-count/index.md.njk

```diff
@@ -80,37 +76,37 @@ Error messages should be styled like this:
 
 {{ example({group: "components", item: "character-count", example: "error", html: true, nunjucks: true, open: false}) }}
 
-If a user tries to submit their response with too many characters, you must show a validation error message above the field in addition to the count message below it.
+If a user tries to send their response with too many characters, you must show an error message above the field as well as the count message below it.
 
-The validation error message tells users what went wrong when they tried to submit their response and how to fix it. The count message provides inline feedback and updates as a user types.
+The error message tells users what went wrong and how to fix it. The count message provides live feedback and updates as a user types.
 
 Make sure errors follow GOV.UK guidance on [writing error messages](https://design-system.service.gov.uk/components/error-message/#be-clear-and-concise) and have specific error messages for specific error states.
 
 #### If the input is empty
 
-Say “Enter [whatever it is]”.<br>
-For example, “Enter a summary”.
+Say ‘Enter [whatever it is]’.<br>
+For example, ‘Enter a summary’.
 
 #### If the input is too long
 
-Say “[whatever it is] must be [number] characters or less”.<br>
-For example, “Summary must be 400 characters or less”.
+Say ‘[whatever it is] must be [number] characters or less’.<br>
+For example, ‘Summary must be 400 characters or less’.
 
 ## Research on this component
 
-This component was originally developed and tested by the Government Digital Service (GDS) in a prototype of the Apply for a temporary event notice service.
+This Government Digital Service (GDS) developed and tested this component in a prototype of the ‘Apply for a temporary event notice’ service.
```

Comment:
```suggestion
The Government Digital Service (GDS) developed and tested this component in a prototype of the ‘Apply for a temporary event notice’ service.
```

Reply:
That's all from me. If you can update this, I'll approve.

Reply:
Done.